fontforge/sfd.c

```diff
@@ -4843,7 +4843,7 @@ return;
     }
     if ( enc>=map->enccount )
 	map->enccount = enc+1;
-    if ( enc!=-1 )
+    if ( enc>=-1 )
```

Comment:
Shouldn't this be `enc>=0`?

Reply:
No, because `.notdef` frequently is U+0000

Reply:
`enc!=-1` is false when enc is -1. `enc>=-1` is true when enc is -1. That's the change I'm asking about.

(Another way of asking: "Shouldn't this be `enc>-1`?")

Reply:
I'm sorry @skef, I'm too tired to answer today, I spent all my energy on a paper about a Unicode character: [[1]](https://raw.githubusercontent.com/ctrlcctrlv/PamudpodUnicodePaper/master/pamudpod.pdf).

Just reading your comment makes me feel like you're right and I'm stupid for (1) even making this mistake and (2) not understanding your comment. Sorry you have to put up with me. :(

Reply:
It's no issue -- this is a primary function of code reviews.